vllm/platforms/cpu.py

```diff
@@ -6,10 +6,10 @@
 class CpuPlatform(Platform):
     _enum = PlatformEnum.CPU
 
-    @staticmethod
-    def get_device_name(device_id: int = 0) -> str:
+    @classmethod
+    def get_device_name(cls, device_id: int = 0) -> str:
         return "cpu"
 
-    @staticmethod
-    def inference_mode():
+    @classmethod
```

Comment:
what's the benefit of this change?

Reply:
It's just to keep it consistent with the others which have been converted into `classmethod`s.

Reply:
I mean, why do we change it to `classmethod`? I think `staticmethod` is sufficient.

Reply:
It is because `has_device_capability` depends on `get_device_capability` of the same class.

Reply:
I see, makes sense.